packages/bezier-react/src/components/Button/Button.tsx

```diff
@@ -113,6 +114,8 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(function Button
   onClick,
   ...rest
 }, forwardedRef) {
+  const Comp = as as 'button'
```

Comment:
타입 에러때문에 단언을 하신걸까요?
```suggestion
  const Comp = as
```
이렇게만 해도 에러가 안뜨네요. 아니면 L101에서 as: Comp = 'button' 으로 해도 될거같아요!

Reply:
에러는 안뜨는데, any로 추론되어 jsx 부분에서 컴파일러가 제대로 동작하지 않는 문제가 있어 추가했어요!